components/post_view/message_attachments/message_attachment/message_attachment.jsx

```diff
@@ -144,14 +148,37 @@ export default class MessageAttachment extends React.PureComponent {
         );
     };
 
-    handleAction = (e) => {
+    handleAction = (e, actionOptions) => {
         e.preventDefault();
+
+        var actionExecutingMessage = this.getActionOption(actionOptions, 'ActionExecutingMessage');
```

Comment:
New to this codebase. But just curious why mixed `var` and `const` together 🤔

Reply:
not part of the codebase yet = ). What is the suggestion?